libraries/AP_RangeFinder/AP_RangeFinder_LeddarOne.cpp

```diff
@@ -27,7 +27,7 @@ extern const AP_HAL::HAL& hal;
 AP_RangeFinder_LeddarOne::AP_RangeFinder_LeddarOne(RangeFinder &_ranger, uint8_t instance,
                                                                RangeFinder::RangeFinder_State &_state,
                                                                AP_SerialManager &serial_manager) :
-    AP_RangeFinder_Backend(_ranger, instance, _state)
+    AP_RangeFinder_Backend(_ranger, instance, _state, MAV_DISTANCE_SENSOR_INFRARED)
```

Comment:
they do ToF calculation with IR led array with 3deg beam, so I though it is more Infrared. I just look on ROS and they use laserscan msg. So I will change for laser